cmd/gobmp/gobmp.go

```diff
@@ -64,6 +65,14 @@ func setupSignalHandler() (stopCh <-chan struct{}) {
 	return stop
 }
 
+func serveHTTP(port int, routes *[]phttp.Route, done <-chan struct{}) error {
```

Comment:
This function be part of the http package rather than this package.

Reply:
Correct, will add in the local package.